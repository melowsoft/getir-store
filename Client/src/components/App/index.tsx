import {Provider} from 'react-redux';
import { store } from '../../state'
import {Container} from "./AppStyles"
import { BrandIcon } from '../BrandIcon';
import { Navbar } from '../Navbar';
import { ContentWrapper } from '../ContentWrapper';
import { ContentArea } from '../ContentArea';


const App = () => {
  return (
  <Provider store={store}>
    <Container>
      <Navbar icon={<BrandIcon />}/>
      <ContentWrapper>
        <ContentArea />
      </ContentWrapper>
    </Container>
  </Provider>
  );
  
}

export default App;
