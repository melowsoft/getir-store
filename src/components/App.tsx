import {Provider} from 'react-redux';
import { store } from '../state'


import {Container} from "./AppStyles"
import { BrandIcon } from './BrandIcon';
import { Navbar } from './Navbar';
import { ContentWrapper } from './ContentWrapper';


const App = () => {
  return <Provider store={store}>
    <Container>
      <Navbar icon={<BrandIcon />}/>
      <ContentWrapper >
        <h1>Hello from Wrapper</h1>
      </ContentWrapper>
    </Container>
  </Provider>;
  
}

export default App;
