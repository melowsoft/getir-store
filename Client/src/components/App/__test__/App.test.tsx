import React from "react"
import  ReactDOM  from "react-dom"
import App from ".."
import { cleanup } from '@testing-library/react';
import renderer from "react-test-renderer"




afterEach(cleanup)

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<App></App>, div);
  ReactDOM.unmountComponentAtNode(div)
})


it("matches snapshot", () => {
  const tree = renderer.create(<App></App>).toJSON()
  expect(tree).toMatchSnapshot()
})