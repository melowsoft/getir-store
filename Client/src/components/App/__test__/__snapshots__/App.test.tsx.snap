// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="sc-bdnxRM khweOe"
>
  <div
    className="sc-hKFxyN duXNjr"
  >
    <div
      className="sc-gtsrHT kcdbmG"
      data-testid="brand-logo"
    >
      <img
        alt="brand-logo"
        className="sc-dlnjwi iWOdBd"
        src="Logo.svg"
      />
    </div>
    <div
      className="sc-eCApnc ebOzOL"
      onClick={[Function]}
    >
      <svg>
        basket.svg
      </svg>
      <p
        className="sc-jSFjdj btJqtq"
      >
        â‚º 
        0
      </p>
    </div>
  </div>
  <div
    className="sc-gKAaRy jMpiJK"
  >
    <div
      className="sc-bBjRSN bRDqDi"
    >
      <div
        className="sc-kLojOw dYCygm"
      >
        <div
          className="sc-bCwfaz"
        >
          <p
            className="sc-fFSPTT ewKuse"
          >
            Sorting
          </p>
          <div
            className="sc-bkbkJK hMkxOU"
          >
            <div
              className="sc-iemWCZ kyDYMU"
            >
              <div
                className="sc-lmgQwP iNLpFY"
              >
                <div
                  className="sc-iwajpm chYJNL"
                  onClick={[Function]}
                >
                  <svg>
                    selected-circle.svg
                  </svg>
                  <p
                    className="sc-cxNHIi jHysL"
                  >
                    Price low to high
                  </p>
                </div>
                <div
                  className="sc-iwajpm chYJNL"
                  onClick={[Function]}
                >
                  <svg>
                    unselected-circle.svg
                  </svg>
                  <p
                    className="sc-cxNHIi jHysL"
                  >
                    Price high to low
                  </p>
                </div>
                <div
                  className="sc-iwajpm chYJNL"
                  onClick={[Function]}
                >
                  <svg>
                    unselected-circle.svg
                  </svg>
                  <p
                    className="sc-cxNHIi jHysL"
                  >
                    New to old
                  </p>
                </div>
                <div
                  className="sc-iwajpm chYJNL"
                  onClick={[Function]}
                >
                  <svg>
                    unselected-circle.svg
                  </svg>
                  <p
                    className="sc-cxNHIi jHysL"
                  >
                    Old to new
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="sc-dIvrsQ gzHxjE"
        >
          <p
            className="sc-fFSPTT ewKuse"
          >
            Brands
          </p>
          <div
            className="sc-bkbkJK dBeLEQ"
            height={230}
          >
            <div
              className="sc-iemWCZ kyDYMU"
            >
              <input
                className="sc-hHEiqL jVgRot"
                onChange={[Function]}
                placeholder="Search Brands"
                type="text"
                value=""
              />
              <div
                className="sc-fKgJPI hZqJLd"
              >
                <div
                  className="sc-dlMDgC fSCzFq"
                  onClick={[Function]}
                >
                  <svg>
                    selected-box.svg
                  </svg>
                  <p
                    className="sc-kfYoZR iGNVed"
                  >
                    ALL (
                    0
                    )
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="sc-bYwzuL caWyjY"
        >
          <p
            className="sc-fFSPTT ewKuse"
          >
            Tags
          </p>
          <div
            className="sc-bkbkJK dBeLEQ"
            height={230}
          >
            <div
              className="sc-iemWCZ kyDYMU"
            >
              <input
                className="sc-hHEiqL jVgRot"
                onChange={[Function]}
                placeholder="Search Tags"
                type="text"
                value=""
              />
              <div
                className="sc-ezzafa fHYJBb"
              >
                <div
                  className="sc-iJCRrE fxhHCm"
                  onClick={[Function]}
                >
                  <svg>
                    selected-box.svg
                  </svg>
                  <p
                    className="sc-giAqHp iqWgwE"
                  >
                    ALL (
                    0
                    )
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="sc-cTJkRt chgna-D"
      >
        <h4
          className="sc-iklJeh evroew"
        >
          Products
        </h4>
        <div
          className="sc-jNnpgg hVNdHU"
        >
          <div
            className="sc-dPaNzc kGqTeg"
            onClick={[Function]}
          >
            Mug
          </div>
          <div
            className="sc-dPaNzc kqTCrY"
            onClick={[Function]}
          >
            Shirt
          </div>
        </div>
        <div
          className="sc-efHYUO hIswCw"
        />
        <div
          className="sc-jJMGnK craUyb"
        >
          <ul
            className="pagination"
          >
            <li
              className="previous disabled"
            >
              <a
                aria-disabled="true"
                aria-label="Previous page"
                className="prev-link-class"
                onClick={[Function]}
                onKeyPress={[Function]}
                rel="prev"
                role="button"
                tabIndex="0"
              >
                <div
                  className="sc-gXfVKN hklMiA"
                >
                  <svg>
                    prev.svg
                  </svg>
                   
                  <div>
                    Prev
                  </div>
                </div>
              </a>
            </li>
            <li
              className="next"
            >
              <a
                aria-disabled="false"
                aria-label="Next page"
                className="next-link-class"
                onClick={[Function]}
                onKeyPress={[Function]}
                rel="next"
                role="button"
                tabIndex="0"
              >
                <div
                  className="sc-hiKfDv ljmpVY"
                >
                  <div>
                    Next
                  </div>
                   
                  <svg>
                    next.svg
                  </svg>
                </div>
              </a>
            </li>
          </ul>
        </div>
      </div>
      <div
        className="sc-dIsUp edCiwp"
        data-testid="basket-box"
      />
    </div>
  </div>
</div>
`;
